src/lv_draw/lv_draw_img.c

```diff
@@ -447,7 +448,7 @@ lv_img_dsc_t *lv_img_buf_alloc(lv_coord_t w, lv_coord_t h, lv_img_cf_t cf)
         lv_mem_free(dsc);
         return NULL;
     }
-    memset(dsc->data, 0, dsc->data_size);
+    memset((uint8_t *)dsc->data, 0, dsc->data_size);
```

Comment:
Is this cast necessary? [`memset`](https://pubs.opengroup.org/onlinepubs/7908799/xsh/memset.html) should take `void *` as the first parameter.

Reply:
`dsc->data` is declared as a `const`, so either that should change or we have to cast it here as not a `const`.

Reply:
> `dsc->data` is declared as a `const`

Makes sense. This is fine.